src/web/components/ArticleHeadlinePadding.tsx

```diff
@@ -26,6 +26,11 @@ const determinePadding = ({
 					padding-bottom: 36px;
 				}
 			`;
+		case Design.LiveBlog:
+		case Design.DeadBlog:
+			return css`
+				padding-top: 3px;
```

Comment:
Not related to this PR but more a question cc @HarryFischer : what are thoughts on changing these paddings to standard Guardian spacing/Source sizes?

Reply:
Actually, that really should be `space.1` I'll fix that!